wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/romireference/commands/TurnDegrees.java

```diff
@@ -0,0 +1,81 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2020 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+/*
+ * Creates a new TurnDegrees command. This command will turn your robot for a 
+ * desired rotation (in degrees) and rotational speed.
+ */
+public class TurnDegrees extends CommandBase {
+  private final Drivetrain m_drive;
+  private final double m_degrees;
+  private final double m_speed;
+  private double startingLeftDistance;
+  private double startingRightDistance;
```

Comment:
These are always 0 do we need them?

Reply:
Good point. These were left over as a workaround before the Encoder.reset fix was put in. I'll remove.

Reply:
Please review again